q2cli/builtin/tools.py

```diff
@@ -290,7 +289,13 @@ def cast_metadata(paths, cast, output_file, ignore_extra,
     with tempfile.NamedTemporaryFile() as temp:
         metadata.save(temp.name)
         cast_md = qiime2.Metadata.load(temp.name, cast_dict)
+
+    if output_file:
         cast_md.save(output_file)
+    else:
+        with tempfile.NamedTemporaryFile(mode='w+') as stdout_temp:
+            cast_md.save(stdout_temp.name)
+            print(stdout_temp.read(), end='')
```

Comment:
After doing some testing on the docs rendering, this actually won't work because it's not truly writing to stdout - so this is still a WIP! Working on this today, and will push up some changes after some more testing.

Reply:
Just pushed up some changes that should resolve this for the docs rendering - will circle back after rendering the docs again to see if that does the trick!